### PR TITLE
Chore/implement hero section

### PR DESCRIPTION
**Pull request by MikeRock51**
image

### CODE REVIEW

**Comment by tech-namulabs** [comment]
# (AI Review) Code Review Results

### Manual Number: 30-2
- src/components/TypingText/TypingText.tsx(False): The code uses a type alias for props, but it does not follow the convention of using `React.ComponentProps<typeof {Component}>` for component props type alias.
- src/components/TypingText/TypingText.tsx(False): The code uses props directly instead of destructuring them, which violates the convention of using props.propsName or {...props.propsName}.

File: src/components/TypingText/TypingText.tsx
Comment (kiteprize) [inline src/components/TypingText/TypingText.tsx:1]: Component Code Must Below Atoms/Component/Modules/Templates
Categorize this Code and Move to below of category

category list
1. Atoms
2. Component
3. Modules
4. Templates

if you don't know how categorize it, check [Convention Docs 20-*](https://coal-blouse-907.notion.site/Namulabs-Work-Convention-783f33a9ec5748d494c0841bf8226c67?pvs=4)

File: src/components/SideBar.tsx/Sidebar.tsx
Comment (kiteprize) [inline src/components/SideBar.tsx/Sidebar.tsx:1]: Component Code Must Below Atoms/Component/Modules/Templates
Categorize this Code and Move to below of category

category list
1. Atoms
2. Component
3. Modules
4. Templates

if you don't know how categorize it, check [Convention Docs 20-*](https://coal-blouse-907.notion.site/Namulabs-Work-Convention-783f33a9ec5748d494c0841bf8226c67?pvs=4)

File: src/components/Atoms/SVGAtom/SkillTagAtom/SkillTagAtom.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import SVGAtom from '../../../Atoms/SVGAtom/SVGAtom'
+
+type Props = {
+  name: string;
+  level: number;
+  icon: "html5" | "bootstrap" | "search";
```
Comment (kiteprize) [inline src/components/Atoms/SVGAtom/SkillTagAtom/SkillTagAtom.tsx:7]: if you use typeof SVGAtom's iconName, use `React.ComponentProps<typeof SVGAtom>['iconName'];`

File: src/components/Modules/Home/HomeContentModule.tsx [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+    <div className='mx-auto max-w-[1300px] py-6'>
+      <div className='mb-8 flex gap-4 text-7xl font-bold'>
+        <h1>Christina</h1>
+        <h1 className='font-outline-2 text-white'>Gray</h1>
+      </div>
+      <div className='grid grid-cols-4 gap-12'>
+        <div className='col-span-1'>
+          <Sidebar />
+        </div>
+        <HeroSectionModule {...props.heroSectionModuleProps } />
```
Comment (kiteprize) [inline src/components/Modules/Home/HomeContentModule.tsx:20]: Don't use Modules in Module code
[Check Module Convention Docs](https://coal-blouse-907.notion.site/20-4-Atomic-Design-Pattern-Module-tsx-11a268a31d0a4a32afdd811b691024a4?pvs=4)

Comment (MikeRock51) [reply src/components/Modules/Home/HomeContentModule.tsx:20]: Thank you so much @kiteprize

**Pull request merged**